addons/website_mail/static/src/xml/website_mail.xml

```diff
@@ -0,0 +1,132 @@
+<templates id="template" xml:space="preserve">
+
+    <t t-name="website_mail.publish_short">
+        <t t-if="is_publisher" t-ignore="true">
+            <div t-attf-class="pull-right js_publish_management #{object.website_published and 'css_published' or 'css_unpublished'}" t-att-data-id="res_id" t-att-data-object="res_model" t-att-data-controller="publish_controller">
+                <button class="btn btn-danger js_publish_btn">Unpublished</button>
+                <button class="btn btn-success js_publish_btn">Published</button>
+            </div>
+        </t>
+    </t>
+
+    <t t-name="website_mail.chatter_message_count">
+        <t t-set="count" t-value="widget.get('message_count')"/>
+        <div class="o_message_counter">
+            <t t-if="count">
+                <span class="o_message_count fa fa-comments"> <t t-esc="count"/></span>
+                <t t-if="count == 1">comment</t>
+                <t t-if="count != 1">comments</t>
+            </t>
+            <t t-if="!count">
```

Comment:
Actually, it might not be. I don't know the status of js qweb. I forgot that subtle distinction.